lib/kuiq/model/process.rb

```diff
@@ -0,0 +1,29 @@
+module Kuiq
+  module Model
+    class ProcessSet
+      def initialize
+        @ps = Sidekiq::ProcessSet.new
```

Comment:
It is easy to forget what `ps` is about, especially that the class itself is also `ProcessSet`, making a developer later on scratch their head while using `ps`. I'd rather clarify what the variable is with a more verbose name that clearly indicates that the class is wrapping a Sidekiq::ProcessSet: `@sidekiq_process_set`